Triphouse-project/index.html

```diff
@@ -8,12 +8,72 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" />
```

Comment:
![2021-04-19_22h20_28](https://user-images.githubusercontent.com/19935417/115291457-77ddff00-a15d-11eb-87eb-a8e2d6774016.png)
Move image to Triphouse-project folder

Reply:
After fix path to image in html and css.

Reply:
Moved img to Triphouse-project folder and fixed path